### PR TITLE
Add controlled and default activeTagIndex, add onActiveTagIndexChange…

### PR DESCRIPTION
**Pull request by bogdanpetru**
… event

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +79,9 @@ Property | Type | Default | Description
 `text` | String | - | the text to show in the filtering input, controlled version of `defaultText`.
 `onTextChange` | Function | - | called when text is changed in the filtering input.
 `disabled` | Boolean | false | prevent all actions on combo: new selection, clear selection.
+`onActiveTagIndexChange(index)` | Function | - | callback called when activeTagIndex changes
```
Comment (radubrehar) [inline README.md]: maybe specify that index can be -1 to denote no tag should be active

File: README.md [hunk 2]
```diff
@@ -79,6 +79,9 @@ Property | Type | Default | Description
 `text` | String | - | the text to show in the filtering input, controlled version of `defaultText`.
 `onTextChange` | Function | - | called when text is changed in the filtering input.
 `disabled` | Boolean | false | prevent all actions on combo: new selection, clear selection.
+`onActiveTagIndexChange(index)` | Function | - | callback called when activeTagIndex changes 
+`activeTagIndex` | Int | - | specify which tag is active, specify it's index, controlled version of `defaultActiveTagIndex`
```
Comment (radubrehar) [inline README.md]: `Number` instead of `Int`

File: src/index.js [hunk 3]
```diff
@@ -50,7 +50,7 @@ export default class Combo extends Component {
       dataMap: {},
       filterValue: '',
       text: props.defaultText || '',
-      activeTagIndex: -1
+      activeTagIndex: props.defaultActiveTagIndex
```
Comment (radubrehar) [inline src/index.js]: ` || -1`?

File: src/index.js [hunk 4]
```diff
@@ -484,7 +486,7 @@ export default class Combo extends Component {
 
     this.prepareValue(props)
 
-    props.activeTagIndex = this.state.activeTagIndex
+    props.activeTagIndex = props.activeTagIndex || this.state.activeTagIndex
```
Comment (radubrehar) [inline src/index.js]: Beware of `0` - in this case, will fallthrough to this.state.activeTagIndex. Better compare with null or undefined

File: src/index.js (hunk 3 above)
Comment (radubrehar) [inline src/index.js]: A, ok, I see defaultActiveTagIndex being -1, that's great

File: src/index.js [hunk 5]
```diff
@@ -636,5 +640,7 @@ Combo.defaultProps = {
   idProperty: 'id',
   displayProperty: 'label',
   disabledProperty: 'disabled',
-  listPosition: 'bottom'
+  listPosition: 'bottom',
+  activeTagIndex: false,
```
Comment (radubrehar) [inline src/index.js]: no need to specify `activeTagIndex` in default props. Or if you do, it's better if it's null or undefined

**Comment by radubrehar** [comment]
Please see my comments. Otherwise, great job!

**Comment by bogdanpetru** [comment]
Thank you, I've added specified improvements

**Comment by bogdanpetru** [comment]
@radubrehar can I accept the pr?

**Comment by radubrehar** [comment]
Yep, sorry for the delay. You can remove `activeTagIndex` from defaultProps.

**Comment by bogdanpetru** [comment]
done, thank you

**Pull request merged**